README.md

```diff
@@ -115,3 +115,25 @@ Example:
 ```bash
 git commit -m "ci(semantic-release): add commitlint and husky as dev tools to write valid commits"
 ```
+
+## Local development
+For local development in a React/Recharts app that uses bento-charts, you can follow these steps for your setup:
+
+1. `build` and `pack` bento-charts
+```bash
+# Builds package and creates a pack file in the "./packs" dir
+npm run buildpack
+```
+
+2. In the project using bento-charts, modify the bento-charts dependency in package.json so that the version number is now the absolute path to the pack file.
+```diff
+- "bento-charts": "2.0.0",
++ "bento-charts": "file:~/bento-charts/packs/bento-charts-2.0.0.tgz",
```

Comment:
does it work with `~`? since that isnt a fully absolute path without expansion

Reply:
Yeah it works!

Reply:
nice

Reply:
I tried using "npm link" for a while to use the local public  container but there was a lot of trouble shooting due to dependencies handling.